dashboard/app/models/peer_review.rb

```diff
@@ -30,14 +30,53 @@ class PeerReview < ActiveRecord::Base
   belongs_to :level
   belongs_to :level_source
 
+  after_save :mark_user_level
+
   REVIEWS_PER_SUBMISSION = 2
+  REVIEWS_FOR_CONSENSUS = 2
 
   enum status: {
     accepted: 0,
     rejected: 1,
     escalated: 2
   }
 
+  def mark_user_level
+    user_level = UserLevel.find_by(user: submitter, level: level)
+
```

Comment:
they don't get deleted in the normal course of operations. I wouldn't preclude having some one-off reason to delete user levels though (CS request, migration...).